src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        Storage.fruitStorage.clear();
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Before
+    public void setUp() {
+        Storage.fruitStorage.put("banana", 80);
+    }
+
+    @Test
+    public void addFruit_AddToDB_Ok() {
```

Comment:
try to name as example below: 
MethodName_StateUnderTest_ExpectedBehavior
in your case you don't need to expain what method do, but expain what you put to the method
```suggestion
    public void addFruit_NotEmptyData_Ok() {
```